src/api/cristinApi.ts

```diff
@@ -121,24 +121,28 @@ export const fetchPerson = async (personId: string) => {
 
 export interface PersonSearchParams {
   name?: string | null;
+  orderBy?: string | null;
   organization?: string | null;
   sector?: string | null;
+  sort?: string | null;
```

Comment:
`asc` trengs kanskje ikke da, siden det er samme som ingen verdi 🤷 
```suggestion
  sort?: 'asc' | 'desc' | null;
```